src/app/features/groups/components/group-modal/group-modal.component.html

```diff
@@ -0,0 +1,16 @@
+<nz-modal [nzTitle]="action==='add'? 'Add new group' : 'Edit selected group'" [nzOkLoading]="isLoading"
```

Comment:
If tag has long list of attributes and inputs it's better to put one in single line

Reply:
Ups! Sorry it is my formmater, changed sorted tags